### PR TITLE
GH-1197 added new screen

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- table view with fake data

### CODE REVIEW

File: SafeUIKit/SafeAppUI/MainFlow/Menu/AddressBook/AddressBookFlowCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+        controller.present(alert, animated: true, completion: nil)
+    }
+
+    func addressBookViewController(controller: AddressBookViewController, edit entry: AddressBookEntry) {
+        // to be replaced with actual implementation
+        print("edit \(entry)")
+    }
+
+    func addressBookViewControllerCreateNewEntry(controller: AddressBookViewController) {
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/Menu/AddressBook/AddressBookFlowCoordinator.swift:32]: why did you change name style compare with the prev. func style?

Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/Menu/AddressBook/AddressBookFlowCoordinator.swift:32]: after refactoring for memory management, I found it helpful to have access to the controller, and this style is more Apple-like style (they pass the delegating class over in the API, too).

**Review by sche — APPROVED** [review]
